src/client/App.tsx

```diff
@@ -21,7 +21,7 @@ export const row = {
 export default function App(): JSX.Element {
   useEffect(() => {
     Object.assign(document.body.style, body)
-  })
+  }, [])
```

Comment:
I would put this change in a `fixup! Update App.tsx to use hooks` commit instead since `Update App.tsx to use hooks` is where the effect hook was added.